src/components/expanded-state/profile/ProfileModal.tsx

```diff
@@ -0,0 +1,168 @@
+import lang from 'i18n-js';
+import React, { useCallback, useRef } from 'react';
+import { View } from 'react-native';
+import Divider from '../../Divider';
+import { ButtonPressAnimation } from '../../animations';
+import { BiometricButtonContent } from '../../buttons';
+import CopyTooltip from '../../copy-tooltip';
+import { Centered, ColumnWithDividers } from '../../layout';
+import { AvatarCircle } from '../../profile';
+import { Text, TruncatedAddress } from '../../text';
+import { ProfileModalContainer, ProfileNameInput } from '.';
+import { useTheme } from '@rainbow-me/context';
+import styled from '@rainbow-me/styled-components';
+import { margin, padding, position } from '@rainbow-me/styles';
+
+const ProfileAddressText = styled(TruncatedAddress).attrs(
```

Comment:
wonder if this new stuff should be design system

Reply:
its not new just copy pasted but yes i concur

Reply:
gonna do this in a new pr just so we can get this merged in